building/building-archlinux-template.md

```diff
@@ -135,11 +140,12 @@ redirect_from:
 
 ## 6:   Run the 'setup' script to build the builder.conf file
 
-*   Run the 'setup' script located in '**/home/user/qubes-builder/**' Make sure you are in directory '**qubes-builder**'
-
-    *   **$ cd /home/user/qubes-builder/**
+*   Run the 'setup' script located in `/home/user/qubes-builder/` Make sure you are in directory `qubes-builder`
```

Comment:
This causes the rest of the list to be mis-formatted:

![screenshot](https://user-images.githubusercontent.com/10417133/47944006-0ba4ad80-dec7-11e8-8097-c5ce31445cfd.png)

Reply:
Thank you, will do.